app/js/components/LabResultsList.jsx

```diff
@@ -138,10 +152,15 @@ export class LabResultsList extends PureComponent {
   handleShowLabTrendsPage(data) {
     const { history } = this.props;
     if (data.order) {
-      history.push({
-        pathname: "/labtrendspage",
-        state: data.encounter.obs[0].concept,
-      });
+      if (data.encounter && data.encounter.obs[0]) {
```

Comment:
Does this assume that there is only one lab result per encounter, and the first obs on the encounter is that result?  I don't know if we necessarily can assume that in all cases...

Reply:
Let's just go ahead and merge this in when you are ready, I can probably comment better once I can test as well...